test/OpenTelemetry.Instrumentation.EntityFrameworkCore.Tests/EntityFrameworkInstrumentationTests.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using Xunit;
+
+namespace OpenTelemetry.Instrumentation.EntityFrameworkCore.Tests;
+
+public class EntityFrameworkInstrumentationTests
+{
+    [Fact]
+    public void ServerAddressWithoutProtocolPrefix()
+    {
+        var activity = new Activity("TestActivity");
+        activity.Start();
+
+        var connection = new
+        {
+            Host = "my.domain.example",
+            DataSource = "tcp:my.domain.example",
+            Port = "5432",
```

Comment:
As I understand, you are DuckTyping some class here. What is your exact type?

Reply:
This question is still valid

Reply:
This file was removed